app/index/reports/controller.js

```diff
@@ -38,10 +38,14 @@ export default Controller.extend({
    * @property {Report[]} reports
    * @public
    */
-  @computed('_allReports.@each.{date,isNew,isDeleted}', 'model')
-  reports(reports, day) {
+  @computed('_allReports.@each.{user,date,isNew,isDeleted}', 'model', 'user')
+  reports(reports, day, user) {
     let reportsToday = reports.filter(r => {
-      return r.get('date').isSame(day, 'day') && !r.get('isDeleted')
+      return (
+        (!r.get('user.id') || r.get('user.id') === user.get('id')) &&
```

Comment:
This is the only place where you also check for `r.get('user.id')` - why?

Reply:
Because this is the only case in which we can have unsaved items, without a user.

Reply:
Ah, i think i also know what part of the UI this belongs to now. Thanks!